tests/e2e/initialization/init_test.go

```diff
@@ -16,11 +16,9 @@ import (
 
 const forkHeight = 10
 
-var (
-	expectedConfigFiles = []string{
-		"app.toml", "config.toml", "genesis.json", "node_key.json", "priv_validator_key.json",
-	}
-)
+var expectedConfigFiles = []string{
+	"app.toml", "config.toml", "genesis.json", "node_key.json", "priv_validator_key.json",
```

Comment:
nit: can we make these multi-line please?

Reply:
I believe this was done automatically through `make format` - I have no preference so happy to change back in this PR but just fyi